build.gradle

```diff
@@ -42,8 +42,13 @@ checkstyle {
     // Keep the below file updated along with subsequent versions of Checkstyle (make sure to choose
     // the tag matching the toolVersion above):
     // https://github.com/checkstyle/checkstyle/blob/checkstyle-10.3.2/src/main/resources/google_checks.xml
-    // NOTE: It may also be necessary to change "${org.checkstyle.google.suppressionfilter.config}"
-    // to "${config_loc}/checkstyle-suppressions.xml" or else the suppressions might be ignored
+    // NOTE: After updating the local version of config/checkstyle/google_checks.xml again in the
```

Comment:
Let's please not lump things together like this. At the very least this should be in its own commit.

Reply:
Noted for the future. We generally squash all our commits within a PR though, so it won't matter in practice.